frontend/app/components/dropdown/dropdown.tsx

```diff
@@ -176,9 +360,31 @@ export default class Dropdown extends Component<Props, State> {
     window.removeEventListener('message', this.receiveMessage);
   }
 
+  onDropdownItemHover(e: Event) {
+    const target = e.target;
+
+    if (target instanceof HTMLElement) {
+      const { id } = target.dataset;
+
+      if (id !== undefined) {
+        this.setState({
+          activeSelectableItemID: +id,
+          isHover: true,
+        });
+      }
+    }
+  }
+
   render() {
-    const { title, titleClass, heading, children, mix, theme } = this.props;
-    const { isActive } = this.state;
+    let { children } = this.props;
```

Comment:
It is not good that you override children depend on props. Now component behavior is not predictable.
For example
```jsx
<Dropdown withSelectableItems=[{title: '2'}]>
  <div>item1</div>
  <div>item2</div>
</Dropdown>
```